.travis.yml

```diff
@@ -33,9 +33,10 @@ install:
   - go get -u github.com/bazelbuild/buildifier/buildifier
 
 script:
-  # Check that all of our samples build
+  # Check that all of our tools and samples build
   - bazel clean && bazel build //...
-  # We have no Bazel-based testing currently.
+  # Check that all of our tests pass
+  - bazel clean && bazel test //...
```

Comment:
I noticed we're doing this in a few other repos, but is there any reason to clean again here between the build and the test?

Reply:
Mostly just that I started from a copy/paste of the line above. :)

I'm happy to change if you feel strongly.